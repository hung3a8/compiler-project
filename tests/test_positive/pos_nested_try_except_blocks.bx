exception DivByZero;
exception OutOfBounds;

def main() {
    try {
        try {
            raise DivByZero;
        } except DivByZero {
            print(42);
            raise OutOfBounds;
        }
    } except OutOfBounds {
        print(43);
    }
    print(1);
}
