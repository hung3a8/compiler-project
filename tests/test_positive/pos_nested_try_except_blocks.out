42
43
1
